config/routes.rb

```diff
@@ -145,8 +145,8 @@
         get 'leaderboard' => 'leaderboard_settings#edit'
         patch 'leaderboard' => 'leaderboard_settings#update'
 
-        get 'topics' => 'discussion/topic_settings#edit', path: 'comments'
-        patch 'topics' => 'discussion/topic_settings#update', path: 'comments'
+        get 'topics' => 'discussion/topic_settings#edit'
+        patch 'topics' => 'discussion/topic_settings#update'
```

Comment:
@allenwq I didn't manage to fix this properly

Reply:
👍  thanks for the info, will double check on this